benches/scheme.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+#[macro_use]
+extern crate bencher;
+pub use std::sync::Arc;
+use bencher::Bencher;
+use cupcake::traits::*;
+
+
+fn encrypt_sk(bench: &mut Bencher) {
+    let fv = cupcake::default();
+
+    let sk = fv.generate_key();
+
+    let mut v = vec![0; fv.n];
```

Comment:
A more Rust-ic way may be
`let mut v = (0..fv.n).map(|i| i as u8).collect::<vec<u8>>();`